lib/systems/doubles.nix

```diff
@@ -94,7 +94,9 @@ in {
                   ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnueabi; })
                   ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnueabihf; })
                   ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnuabin32; })
-                  ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnuabi64; });
+                  ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnuabi64; })
+                  ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnuabielfv1; })
+                  ++ filterDoubles (matchAttrs { kernel = parse.kernels.linux; abi = parse.abis.gnuabielfv2; });
```

Comment:
Something tells me we're going to get burned by having both `abis.gnuabi64`, `gnuabielfv1`, and `gnuabielfv2`.  One of them is always redundant, but which one is redundant depends on endianness (ugh).

Can we get away with dropping `gnuabi64`?  The only situation where this would be a problem would be if we needed to convert an `abi` to an autoconf-name-fragment without having the rest of the `hostPlatform` around... but I don't think we ever need to do that.

Reply:
> Can we get away with dropping `gnuabi64`?

I meant to write:

> Can we get away with dropping `gnu` on `powerpc64be`?

Reply:
My original thought was that `gnuabielfv1` is for ELFv1, `gnuabielfv2` is for ELFv2, and `gnu` is for "whatever the default in GCC defines", but after reflection, I think it doesn't add any value, and I think right in that it just adds confusion. I'll do a "gnu abi is ambiguous on ppc64" error message, like it's done for some arm systems.